test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/DownloadCountToVisibilityConverterTests.cs

```diff
@@ -10,9 +10,9 @@ namespace NuGet.PackageManagement.UI.Test.Converters
     public class DownloadCountToVisibilityConverterTests
     {
         [Theory]
-        [InlineData(0)]
-        [InlineData(1)]
-        [InlineData(2147483657)]
+        [InlineData(0L)]
+        [InlineData(1L)]
+        [InlineData(2147483657L)]
```

Comment:
Added `L` at the end explicitly due to https://github.com/xunit/xunit/issues/2572